src/lang/lang_parser.mly

```diff
@@ -243,11 +244,10 @@ g: | {} | GETS {}
 
 exprs:
   | expr s                   { $1 }
-  | expr exprs               { mk ~pos:$loc (Seq ($1,$2)) }
-  | expr SEQ exprs           { mk ~pos:$loc (Seq ($1,$3)) }
+  | expr s exprs             { mk ~pos:$loc (Seq ($1,$3)) }
   | binding s                { mk_let ~pos:$loc($1) $1 (mk ~pos:$loc unit) }
-  | binding exprs            { mk_let ~pos:$loc($1) $1 $2 }
-  | binding SEQ exprs        { mk_let ~pos:$loc($1) $1 $3 }
+  | binding s exprs          { mk_let ~pos:$loc($1) $1 $3 }
+  | OPEN expr s exprs        { mk ~pos:$loc (Open ($2,$4)) }
```

Comment:
Should we restrict `OPEN` to `var` only?

Reply:
Nope, we want to be able to open `a.b` for instance, so let's not add arbitrary restrictions.